src/components/Table/TableBody.tsx

```diff
@@ -0,0 +1,19 @@
+import { ReactNode } from 'react'
+
+interface TableBodyProps {
+  className?: string
+  children: ReactNode
+}
+
+export function TableBody (props: TableBodyProps): JSX.Element {
+  const { className = '', children } = props
+  return (
+    <div className={`table-body flex flex-col ${className}`}>
+      {children}
+    </div>
+  )
+}
+
+TableBody.defaulProps = {
+  className: ''
+}
```

Comment:
remove this and use classNames 

https://github.com/DeFiCh/scan/blob/b93efa3cec97a469ad5ce499fa6b8ae75ff822ee/src/components/commons/AdaptiveTable.tsx#L23

Reply:
Deleted this component.